### PR TITLE
Update microsoft-365-business-faqs.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected answer as the CUstomer can add Business voice

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2163

**Comment by msbemba** [comment]
@JohanFreelancer9  please copy review

File: microsoft-365/business/support/microsoft-365-business-faqs.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ Windows AutoPilot is a service that streamlines the deployment of new Windows 10
...
  
 ### Can I add Phone System and Calling Plans to Microsoft 365 Business Premium? 
-No, Phone System and Calling Plan are reserved for customers who have more advanced needs. Customers who require these capabilities should look at Microsoft 365 Enterprise offerings. 
+Yes, You can purchase Business Voice Subscription. [Business Voice](../business-voice/whats-business-voice.md) is a telephony solution designed for small and medium businesses of up to 300 users that bundles Phone System, Audio Conferencing, a Domestic Calling Plan, and more. Phone System capabilities that you'll get include voicemail, caller ID, call park, call forwarding, auto attendants, and call queues. <br> To find out whether Business Voice is available in your country or region, see [Country and region availability for Business Voice](../business-voice/country-region-availability.md). For pricing info, see [Pricing for Business Voice](https://go.microsoft.com/fwlink/?linkid=2127221).
```
Comment (JohanFreelancer9) [inline microsoft-365/business/support/microsoft-365-business-faqs.md]:
```suggestion
Yes, you can purchase a Business Voice subscription. [Business Voice](../business-voice/whats-business-voice.md) is a telephony solution designed for small and medium businesses of up to 300 users that bundles Phone System, Audio Conferencing, a Domestic Calling Plan, and more. Phone System capabilities that you'll get include voicemail, caller ID, call park, call forwarding, auto attendants, and call queues. <br> To find out whether Business Voice is available in your country or region, see [Country and region availability for Business Voice](../business-voice/country-region-availability.md). For pricing information, see [Pricing for Business Voice](https://go.microsoft.com/fwlink/?linkid=2127221).
```

Comment (JohanFreelancer9) [reply microsoft-365/business/support/microsoft-365-business-faqs.md]: Please change capitalization in the first sentence as indicated, and change "info" to "information" in the last sentence.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**